cmd/ipfs-cluster-follow/commands.go

```diff
@@ -486,6 +486,57 @@ func listCmd(c *cli.Context) error {
 	return nil
 }
 
+// Stop
+func stopCmd(c *cli.Context) error {
+	clusterName := c.String(clusterNameFlag)
+
+	absPath, configPath, identityPath := buildPaths(c, clusterName)
+	if !isInitialized(absPath) {
+		printNotInitialized(clusterName)
+		return cli.Exit("", 1)
+	}
+
+	// unpin cluster files and kill ipfs-cluster-follow process if cluster and ipfs daemon exist
+	_, err := os.Stat(filepath.Join(absPath, "api-socket"))
+	if err == nil {
+		err = unpinEverything(absPath, clusterName, identityPath)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "unpin error"), 1)
+		}
+		err = killFollower(absPath)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "error killing follower peer(ipfs-cluster-follow daemon)"), 1)
+		}
+	} else {
+		return cli.Exit(errors.Wrap(err, "ipfs-cluster-follow or ipfs daemon not exists"), 1)
```

Comment:
This would be more like `ipfs-cluster-follow does not seem to be running for this cluster`. ?

Reply:
sound great